.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
Hi 
we dont need this file. 
local server pert for VS Code Live server is pretty much individual
could you please either delete this file or add into exclusion with [.gitignore](https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/)